webplugin/js/app/zendesk-chat-service.js

```diff
@@ -137,3 +137,17 @@ function ZendeskChatService() {
        
     }
 };
+
+
+var onTabClickedHandlerForZendeskConversations = function (event) {
+    console.log("onTabClicked from zendesk: ", event, MCK_GROUP_MAP[event.tabId]);
+    if (kommunicate._globals.zendeskChatSdkKey) {
+        let currentGroupData = MCK_GROUP_MAP[event.tabId];
```

Comment:
please use var, let wouldn't work in some browsers

Reply:
done